.github/workflows/nctl-scan-repo.yaml

```diff
@@ -0,0 +1,51 @@
+name: NCTL Scan Terraform Demo
+run-name: ${{ github.actor }} has triggered Scan Action 🚀
+on:
+  pull_request:
+    branches:
+      - "main"
+  push:
+    branches:
+      - "main"
+env:
+  AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY_ID}}
+  AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+  AWS_SESSION_TOKEN: ${{secrets.AWS_SESSION_TOKEN}}
+
+jobs:
+  NCTL-Scan-Terraform:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+
+      - name: Check out repository code
+        uses: actions/checkout@v4
+
+      - name: NCTL-scan-installer - Terraform
+        uses: supplypike/setup-bin@v4
+        with:
+          uri: 'https://nirmata-downloads.s3.us-east-2.amazonaws.com/nctl/nctl_4.1.2/nctl_4.1.2_linux_386.zip'
+          name: 'nctl'
+          version: '4.1.2'
+
+      - name: Check nctl version
+        run: |
+          nctl version
+          nctl login --url https://nirmata.io --userid dolis@nirmata.com --token 87s9aymmWxG1c8q9min2caU6M4X9kmDtXQNc+LFQHLLCVhsW6QzHawQ2pQKZr5e3xc4zgTm+9cU3tl1v+d5WbA==
```

Comment:
Should probably ENV var this at the very least :)

Reply:
Thanks, B! I have invalidated the key.